src/main/java/com/openshift/internal/restclient/http/UrlConnectionHttpClientBuilder.java

```diff
@@ -44,6 +48,12 @@ public UrlConnectionHttpClientBuilder setCredentials(String username, String pas
 		return setAuthorizationStrategy(new BasicAuthorizationStrategy(username, password, token));
 	}
 	
+	public UrlConnectionHttpClientBuilder setCertificate(String alias, X509Certificate cert) {
```

Comment:
My knowledge of certs is limited so I'm just asking, would we always expect this to be pairs of alias/cert?  opposed to providing 2 separate builder methods.  Also, is there any thought if there is need or if it makes sense for a consumer to call this multiple times for more then one cert?